### PR TITLE
feat(anthos): unprod anthos product

### PR DESCRIPTION
**Pull request by mohammed-zahaf**
| Question         | Answer
| ---------------- | ---
| Branch?          | `release/component-ww-14`
| Bug fix?         | no
| New feature?     | no
| Breaking change? | no
| Tickets          | Feat #MANAGER-11095
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~

## Description
1. Remove Anthos Product App

### CODE REVIEW

File: packages/manager/apps/dedicated/client/app/account/user/components/newAccountForm/field/new-account-form-field-component.controller.js [hunk 1]
```diff
@@ -6,11 +6,15 @@ import map from 'lodash/map';
...
-  PHONE_PREFIX,
-  MODEL_DEBOUNCE_DELAY,
   FIELD_NAME_LIST,
+  MODEL_DEBOUNCE_DELAY,
+  PHONE_PREFIX,
 } from '../new-account-form-component.constants';
 
+const a = 1;
+const b = a + FIELD_NAME_LIST.address;
+const c = `${a + FIELD_NAME_LIST.address}À`;
```
Comment (anooparveti) [inline packages/manager/apps/dedicated/client/app/account/user/components/newAccountForm/field/new-account-form-field-component.controller.js:16]: **nitpick:** Looks like an unnecessary change

**Review by anooparveti — CHANGES_REQUESTED** [review]

**Review by ghyenne — APPROVED** [review]

File: packages/manager/apps/dedicated/client/app/account/user/components/newAccountForm/field/new-account-form-field-component.controller.js (hunk 1 above)
Comment (mohammed-zahaf) [reply packages/manager/apps/dedicated/client/app/account/user/components/newAccountForm/field/new-account-form-field-component.controller.js:16]: ah my bad, not required.
I was made some test and it was committed within.

**Review by ghyenne — DISMISSED** [review]

**Review by anooparveti — DISMISSED** [review]
